frontend/test/unit/specs/utils/search-query-transform.spec.js

```diff
@@ -247,8 +249,12 @@ describe("searchQueryTransform", () => {
       searchBy: [
         { code: "creator", checked: true, name: "filters.searchBy.creator" },
       ],
-      mature: [
-        { code: "mature", name: "filters.mature.mature", checked: true },
+      includeSensitiveResults: [
+        {
+          code: "includeSensitiveResults",
+          name: "filters.mature.mature",
```

Comment:
Does this name need to change as well?

Reply:
Since this PR does not change the string, it's better not to rename this key, too. Otherwise, we will make this string "fuzzy" in GlotPress - which is not really necessary just yet.